packages/server-core/src/user/user/user.hooks.ts

```diff
@@ -82,12 +82,15 @@ const restrictUserPatch = async (context: HookContext<UserService>) => {
   if (loggedInUser.id !== context.id)
     throw new Error("Must be an admin with user:write scope to patch another user's data")
 
-  // If a user without admin and user:write scope is patching themself, only allow changes to avatarId and name
+  // If a user without admin and user:write scope is patching themself, only allow changes to specific properties
```

Comment:
Since we are not longer using remove method of user service. Therefore should remove it from `userMethods` array in schema file and hooks?